1080i/Includes_View_54_AEL.xml

```diff
@@ -167,4 +178,31 @@
             <include>View_540_List_Info</include>
         </control>
     </include>
+
+    <include name="View_541_Wall_Info">
+        <control type="group">
+            <include>View_Group</include>
+            <visible>Control.IsVisible(541)</visible>
+            <include content="View_51_Wall_Container">
+                <param name="id" value="541" />
+                <param name="height" value="view_height" />
+                <param name="viewtype" value="$LOCALIZE[31437]" />
+                <param name="viewtypename" value="icon" />
+                <param name="controllayout" value="View_514_Wall_Info_Layout" />
+                <param name="forced" value="true" />
+                <param name="visible" value="Container.Content(movies) | Container.Content(sets) | Container.Content(tvshows) | Container.Content(seasons) | $EXP[Exp_IsPluginAdvancedLauncher]" />
```

Comment:
Shouldn't this only be visible for AEL and not movies/sets/tvshows? Seems like it would be confusing to have two very similar views available for movies. 

I think it might make more sense to have this new view *only* available for AEL, and then have the standard version *only* available for movies/sets/tvshows. Unless there's something I'm misunderstanding here?

Reply:
You are 100% right. That was a copy-pasta mistake on my part. Changed to be exclusive for AEL.